mchf-eclipse/.settings/ilg.gnumcueclipse.managedbuild.cross.prefs

```diff
@@ -0,0 +1,2 @@
+buildTools.path=C\:\\Program Files\\GNU MCU Eclipse\\Build Tools\\2.10-20180103-1919\\bin
+eclipse.preferences.version=1
```

Comment:
Guys, Hi... Could you avoid putting your local eclipse settings into REPO. Thank you..

Reply:
I have merged, removed the files and pushed back to repo.

Reply:
Thanks Andreas!